graphviz build failure when libtool-ltdl-devel is missing on Fedora

### Operating system

Linux Fedora

### Compiler

gcc

### Steps to reproduce the behavior

```Shell
Run `vcpkg install graphviz` and the build will fail if you don't have libtool-ltdl-devel installed.

there is a hint for ubuntu users already, namely:
 "graphviz currently requires the following libraries from the system package manager:
    libtool
They can be installed with apt-get install libtool"

I think that hint should be improved to say something like:
"They can be install with `apt-get install libtool` on Ubuntu systems or `dnf install libtool-ltdl-devel` on Fedora systems"
```


### Failure logs

```[1085]$ less /home/allen/projects/vcpkg/buildtrees/graphviz/config-x64-linux-dbg-err.log
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find LTDL (missing: LTDL_LIBRARY LTDL_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindLTDL.cmake:8 (find_package_handle_standard_args)
  /home/allen/projects/vcpkg/scripts/buildsystems/vcpkg.cmake:859 (_find_package)
  CMakeLists.txt:120 (find_package)
```

### Additional context

_No response_

--- a/ports/graphviz/portfile.cmake
+++ b/ports/graphviz/portfile.cmake
@@ -19,7 +19,7 @@ vcpkg_from_gitlab(
 if(VCPKG_TARGET_IS_OSX)
     message("${PORT} currently requires the following libraries from the system package manager:\n    libtool\n\nThey can be installed with brew install libtool")
 elseif(VCPKG_TARGET_IS_LINUX)
-    message("${PORT} currently requires the following libraries from the system package manager:\n    libtool\n\nThey can be installed with apt-get install libtool")
+    message("${PORT} currently requires the following libraries from the system package manager:\n    libtool\n\nThey can be install with `apt-get install libtool` on Ubuntu systems or `dnf install libtool-ltdl-devel` on Fedora systems")
 endif()
 
 vcpkg_list(SET OPTIONS)

--- a/ports/graphviz/vcpkg.json
+++ b/ports/graphviz/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "graphviz",
   "version-semver": "9.0.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Graph Visualization Tools",
   "homepage": "https://graphviz.org/",
   "license": "EPL-1.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3062,7 +3062,7 @@
     },
     "graphviz": {
       "baseline": "9.0.0",
-      "port-version": 1
+      "port-version": 2
     },
     "greatest": {
       "baseline": "1.5.0",

--- a/versions/g-/graphviz.json
+++ b/versions/g-/graphviz.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "83813d9389be10cbd741a2e6dbd32924972e6b82",
+      "version-semver": "9.0.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "ba712eb597e2f6b4cd9113689ccb72cb95025d6f",
       "version-semver": "9.0.0",
       "port-version": 1

